plugin-hrm-form/src/components/case/timeline/Timeline.tsx

```diff
@@ -188,7 +188,9 @@ const Timeline: React.FC<OwnProps> = ({
             if (timelineActivity.isDraft) {
               canViewActivity = false;
             } else {
-              canViewActivity = can(PermissionActions.VIEW_CONTACT, timelineActivity);
+              const contactId = timelineActivity.activity.id;
```

Comment:
```suggestion
             canViewActivity = can(PermissionActions.VIEW_CONTACT, timelineActivity.activity)
```
The above change should be all that's required? The contact activities in `timelineActivities` would contain the complete contact, not just the id